model/trace/rpc.yaml

```diff
@@ -71,6 +79,12 @@ groups:
       - ref: rpc.grpc.response.metadata
         tag: grpc-tech-specific
         requirement_level: opt_in
+      - ref: rpc.request.body.size
+        tag: grpc-tech-specific
+        requirement_level: recommended
```

Comment:
```suggestion
        requirement_level: recommended
        note: >
            The request body size represents the serialized size of the message payload that is sent over the wire. In case of streaming calls, it accounts for the original message in the request and does not include size of the consequent message stream.
             This value can usually be obtained from protobuf API such as [AbstractMessage#getSerializedSize](https://protobuf.dev/reference/java/api-docs/com/google/protobuf/AbstractMessage.html#getSerializedSize--).
```